features/proxy_config.feature

```diff
@@ -13,23 +13,18 @@ Feature: Proxy configuration
             dns_v4_first on\nacl all src 0.0.0.0\/0\nhttp_access allow all
             """
         And I run `systemctl restart squid.service` `with sudo` on the `proxy` machine
-        When I create the file `/etc/ubuntu-advantage/uaclient.conf` with the following:
+        And I run `ua config set https_proxy=http://<ci-proxy-ip>:3128` with sudo
+        Then stdout matches regexp:
         """
-        contract_url: 'https://contracts.canonical.com'
-        data_dir: /var/lib/ubuntu-advantage
-        log_level: debug
-        log_file: /var/log/ubuntu-advantage.log
-        ua_config:
-          http_proxy: http://<ci-proxy-ip>:3128
-          https_proxy: http://<ci-proxy-ip>:3128
+        Setting snap proxy
         """
-        And I verify `/var/log/squid/access.log` is empty on `proxy` machine
-        # We need this for the route command
-        And I run `apt-get install net-tools` with sudo
-        # We will guarantee that the machine will only use the proxy when
-        # running the ua commands
-        And I run `route del default` with sudo
-        And I attach `contract_token` with sudo and options `--no-auto-enable`
+        When I run `ua config set http_proxy=http://<ci-proxy-ip>:3128` with sudo
+        Then stdout matches regexp:
+        """
+        Setting snap proxy
+        """
+        # TODO No longer empty, needs researching
```

Comment:
There was a bug in the "verify `file` is empty" step that was fixed in #1979 
You may need to add a line with this
```
        When I run `truncate -s 0 /var/log/squid/access.log` `with sudo` on the `proxy` machine
```
Early in the test to make sure the log starts out empty

Reply:
Yeah, It kept failing here, I thought it was my environment setup.

Reply:
The PR you referenced only shows the inclusion of options --no-auto-enable. Is that needed as well?

Reply:
@CalvoM I shouldve been more specific. This line is what I was referring to that affects the `I verify file is empty` step https://github.com/canonical/ubuntu-advantage-client/pull/1979/files#diff-cf027576a6812e3403bd3d9579d83abfaa74a41607a4f3c986398ef8492a5d3dR196

I'm not sure if you need `--no-auto-enable` here or now. If it works without it, then great :)

Reply:
Oh yeah. However, when setting the proxy via the cli will have some entries in the access.log file since `validate_proxy` makes some requests via the proxy. So, instead of testing if it empty, we could test that the test urls are in the log file.